lightning/src/ln/channelmanager.rs

```diff
@@ -2061,7 +2061,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		};
 
 		for htlc_source in failed_htlcs.drain(..) {
-			self.fail_htlc_backwards_internal(self.channel_state.lock().unwrap(), htlc_source.0, &htlc_source.1, HTLCFailReason::Reason { failure_code: 0x4000 | 8, data: Vec::new() });
+			self.fail_htlc_backwards_internal(self.channel_state.lock().unwrap(), htlc_source.0, &htlc_source.1, HTLCFailReason::Reason { failure_code: 0x4000 | 8, data: Vec::new() }, self.get_our_node_id());
```

Comment:
I'm having trouble trying to parse what the call-sites of `self.fail_htlc_backwards_internal` are actually doing. So, for now, I pass in our `node_id` to return as a the `sink_node_id`. I need some help here.